AlertaDengue/dados/templates/main.html

```diff
@@ -1,224 +1,104 @@
 {% extends 'base.html' %}
 {% load i18n %}
 {% load static %}
+{% load home_components %}
 {% load lookup %}
-{% block extra_head %}
-    <meta property="og:image"
-          content="http://info.dengue.mat.br/static/img/screenshot.png"/>
-    <meta property="og:image:secure_url"
-          content="https://info.dengue.mat.br/static/img/screenshot.png"/>
-    <style>
```

Comment:
In the future we should keep CSS in separate files, but that should be done on a new issue